src/client/components/ReviewEditorComponent/ReviewEditorComponent.js

```diff
@@ -0,0 +1,64 @@
+import React ,{useState}  from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button/Button';
+import unknownUser from '../../assets/images/portraits/unknown-black.jpg';
+import './ReviewEditorComponent.style.css';
+
+export default function ReviewEditorComponent({
+  avatarUrl,
+  fullName,
+  inputText,
+  placeHolderText,
+  buttonStyle,
+}) {
+  const [inputValue,setInputValue] = useState()
+  const onChange = (event) => {
+    setInputValue((event.target.value))
+  };
+  return (
+    <div className="container">
+      <div className="top-part">
+        <div className="profile-picture">
+          <div className="circle-mask">
+            <img
+              src={avatarUrl || unknownUser}
+              alt="Portrait of user"
```

Comment:
@pdsorensen For some reason the "unknownUser" doesn't show the picture! It suppose to bring a fallback pic in case user doesn't have one. Can be seen in the second scenario of Storybook.

Reply:
Your image import should be `import unknownUser from '../../assets/images/portraits/unknown-black.jpg';`

Reply:
Sorry, edited it to be correct. It should be png.

Reply:
Unfortunately no succeed, have checked beforehand!

Reply:
Keep debugging @arya56. I believe you can do it. :)

Reply:
@pdsorensen still no success here.

Reply:
solved, with last update loader dependency.